test/betting/helpers/bootstrap.jsligo

```diff
@@ -34,13 +34,12 @@ export const bootstrap = () => {
   };
 
   // Boostrapping BETTING contract
-  let betting_path = "../../../src/contracts/jsligo/betting/main.jsligo";
-  let iBis = Test.run( (x : Types.storage) => x, init_storage);
-  let [betting_address, _c, _n] = Test.originate_from_file(betting_path, iBis, 0 as mutez);
-  let betting_taddress = (Test.cast_address(betting_address) as typed_address<parameter_of Betting,Types.storage> );
-  let betting_contract = Test.to_contract (betting_taddress);
+  const betting_path = "../../../src/contracts/jsligo/betting/main.jsligo";
+  const orig : origination_result<parameter_of Betting,Types.storage> = Test.originate_from_file(betting_path, init_storage, 0 as mutez);
+  const betting_contract = Test.to_contract (orig.addr);
+  const betting_address = Test.to_address (orig.addr);
```

Comment:
:thinking: isn't orig.addr already an address? If not maybe we should fix the API, sounds confusing

Reply:
it's a "typed address".. but kinda annoying to have such a long label for it